package/translate/he.po

```diff
@@ -167,7 +167,7 @@ msgstr "גובה שדה החיפוש"
 
 #: ../contents/ui/config/ConfigGeneral.qml
 msgid "Windows (White)"
-msgstr "Windows ‏(לבן)"
+msgstr "‏Windows (לבן)"
```

Comment:
What changed on this line?

Reply:
Looks like previously, `e280 8f` was before the bracket. It has now been moved to before "Windows".

```
00000000:  6d73 6773 7472 2022 5769 6e64 6f77 7320 e280 8f28 d79c d791  :msgstr "Windows ...(....
00000018:  d79f 2922 0a6d 7367 7374 7220 22e2 808f 5769 6e64 6f77 7320  :..)".msgstr "...Windows 
00000030:  28d7 9cd7 91d7 9f29 22                                       :(......)"
```

Reply:
The direction character was moved to the start of the string in order to align the string from right.
![the-change](https://user-images.githubusercontent.com/66558205/114979881-4848a180-9e94-11eb-961b-71a4537c885b.png)